create/how_to/qovery.md

```diff
@@ -1,38 +1,63 @@
 # Deploy a MeiliSearch instance on Qovery
 
-_The following is a guest post by Romaric Philogène, CEO and Co-Founder of Qovery._
+[Qovery](https://www.qovery.com) is a fully-managed cloud platform that runs on your AWS, Digital Ocean and Scaleway account where you can host static sites, backend APIs, databases, cron jobs, and all your other apps in one place.
 
-## Introduction
+Qovery provides **free hosting** for individual developers and include the following features:
 
-In this tutorial I explain how to deploy a pre-configured Meilisearch instance in one click.
+* Continuous, automatic builds & deploys from GitHub and GitLab.
+* Automatic SSL certificates through [Let's Encrypt](https://letsencrypt.org).
+* Free SSD storage.
+* Unlimited collaborators.
+* Unlimited [custom domains](https://hub.qovery.com/docs/using-qovery/configuration/application/#domains).
 
-*Qovery provides free Cloud hosting with databases, SSL, a global CDN, and automatic deploys with Git.*
+## Setup
 
-Deploying Meilisearch with Qovery provides:
+### 1. Create a Qovery account
 
-- A pre-configured Meilisearch instance.
-- A free SSD storage.
-- A free SSL.
-- Optional: custom domain.
+Visit the [Qovery dashboard](https://start.qovery.com) to create an account if you don't already have one.
 
-## Step-by-step MeiliSearch deployment
+### 2. Create a project
+* Click on **Create project** and give a name to your project.
+* Click on **Next**.
```

Comment:
### 2. Create a project

* Click on **Create project** and give a name to your project
* Click on **Next**